docker-entrypoint

```diff
@@ -56,6 +58,21 @@ restart_container() {
   docker_curl -f -X POST "${HTTP_ENDPOINT}/containers/${container_id}/restart?t=${timeout}"
 }
 
+notify_webhook() {
+  local text="$1"
+  local data="{\"text\": \"${text}\"}"
+
+  if [ -n "$WEBHOOK_SLACK" ]
+  then
+    curl -X POST -H "Content-type: application/json" -d $data  $WEBHOOK_SLACK
+  fi
+  
+  if [ -n "$WEBHOOK_TEAMS" ]
+  then
+    curl -X POST -H "Content-Type: application/json" -d $data $WEBHOOK_TEAMS
+  fi
+}
```

Comment:
Having both `WEBHOOK_TEAMS` and `WEBHOOK_SLACK` seems a bit redundant. Especially since the curl call is exactly the same.

Reply:
Yeah. I was thinking the same at some point. But I was thinking in the longer term we could use different json payloads for nicer alerts (e.g. https://docs.microsoft.com/en-us/microsoftteams/platform/webhooks-and-connectors/how-to/connectors-using). But if we just want to send plain alert messages, you are totally right.

Reply:
basic plain text is good to starts with, however very likely next feature request would be to add/amend information on notification or like you said prettify. 
commonly used then are templates, which user can customise for what type of container's information they want in notification along with however they'd like to style it.
Example https://github.com/hasnat/docker-events-notifier/tree/master/templates

Reply:
Yeah that would be cool. But I would suggest to start with a MVP and enhance this (maybe also e-mail support) when adoption kicks in.